boards/nucleo144-common/include/gpio_params.h

```diff
@@ -34,10 +32,25 @@ extern "C" {
 static const  saul_gpio_params_t saul_gpio_params[] =
 {
     {
-        .name = "LED(green)",
+        .name = "LD1(green)",
         .pin = LED0_PIN,
         .mode = GPIO_OUT
     },
+    {
+        .name = "LD2(blue)",
+        .pin = LED1_PIN,
+        .mode = GPIO_OUT
+    },
+    {
+        .name = "LD3(red)",
+        .pin = LED2_PIN,
+        .mode = GPIO_OUT
+    },
+    {
+        .name = "B1(User button)",
+        .pin = BTN_B1_PIN,
+        .mode = GPIO_IN_PU
```

Comment:
On nucleo144, ti pin is pulled-down externally, so it should be either `GPIO_IN` or `GPIO_IN_PD`